### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by mrVin99** [konform-kt/konform#131] [General]
I'm unable to install and use this library. I did the following, added it to dependencies.
```
dependencies {
    implementation("io.konform:konform-jvm:0.6.2")
    testImplementation(kotlin("test"))
}
```

And when trying to use it following the exact demo example, it cannot be found.
![image](https://github.com/user-attachments/assets/2c053d0b-4826-46ad-8367-e76a8b874a0c)

When trying to force compilation, i get
```
What went wrong:
Execution failed for task ':compileKotlin'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find io.konform:konform-jvm:0.6.2.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/io/konform/konform-jvm/0.6.2/konform-jvm-0.6.2.pom
     Required by:
         project :
```

It seems it cannot find the maven repository to download it, any help?

**Comment by dhoepelman** [comment]
Hi, please use version 0.6.1.

Version 0.6.2 is not released yet due to some changes in how publishing to maven central works, which I haven't had time to look into yet.

**Reply by ghost** [reply]
Thanks

**Comment by dhoepelman** [comment]
0.6.2 is released now
